openqabot/args.py

```diff
@@ -233,6 +240,51 @@ def get_parser():
     )
     cmdaggrsync.set_defaults(func=do_sync_aggregate_results)
 
+    cmdincrementapprove = commands.add_parser(
+        "increment-approve",
+        help="Approve the specified product increment if tests passed",
+    )
+    cmdincrementapprove.add_argument(
+        "--obs-project",
+        required=False,
+        type=str,
+        default="SUSE:SLFO:Products:SLES:15.99:TEST",
```

Comment:
I found it a bit weird that the parameter is implies the build.opesuse.org but the default project cant be found there. I guess just `--project` would do better fit. wdyt?

Reply:
But the OBS prefix makes it clear that this relates to the configured OBS instance (which can of course be our internal one) and not to some of the other services (like SMELT, the dashboard and openQA).

Note that OBS is the name of the software, not the name/domain of a specific instance of it (like build.opensuse.org).